docs: add org README (#16)
aws-deadline/.github: profile/README.md

# AWS Deadline Cloud GitHub Org

Home for open source code related to AWS Deadline Cloud.

## Quick start

**If you want to install Deadline Cloud tools on your workstation**, you can run the Deadline Cloud submitter installer. The installer can be found by going to the AWS Console > AWS Deadline Cloud > Downloads > Deadline Cloud submitter installer > Download.

**If you're looking for documentation for the AWS Deadline Cloud service**, see the [AWS Deadline Cloud documentation](https://docs.aws.amazon.com/deadline-cloud/).

**If you encounter a bug with an integration or tool**, please open an issue in the relevant repository.

> **If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for  example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples. 

Reviewer: ```suggestion
**If you want to build your own integration**, you can look through [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) for example job bundles, conda recipes, and other tooling. You can also look through the pre-built integrations below for larger examples. 
```

> **If you're looking for more information Open Job Description which Deadline uses for defining jobs**, see the [Open Job Description specification](https://github.com/OpenJobDescription/openjd-specifications/) or other repositories in the [Open Job Description GitHub org](https://github.com/OpenJobDescription/).

Reviewer: ```suggestion
**If you're looking for more information on defining jobs**, see the [Open Job Description specification](https://github.com/OpenJobDescription/openjd-specifications/) or other repositories in the [Open Job Description GitHub org](https://github.com/OpenJobDescription/).
```

> **If you want to contribute to an integration or library**, contributions are gratefully accepted!  You can open feature request issue to request a feature or to propose early feedback on an idea before raising a pull requests. Pull requests are welcomed  for features that would be widely useful to the community and which align with the existing codebase.

Reviewer: nit: was a little word salad-y. Here's my attempt
```suggestion
**If you want to contribute to an integration or library**, contributions are gratefully accepted! You can open an issue to request a feature or receive early feedback on an idea before raising a pull request. Pull requests are welcomed for features that would be widely useful to the community and which align with the existing code base.
```

**If you want to adapt an existing integration to your specific workload**, consider forking a repository and making your changes in the fork.

## Integrations
Many repositories in this GitHub org are integrations with digital content creation software. See documentation in the individual repos for more info about features, usage, and compatability.

Deadline Cloud supports service-managed fleets (SMF) and customer-managed fleets (CMF). With SMF, 

> Deadline fully manages the workers running the jobs and includes some pre-configured software. With CMF, the workers run on any customer-managed compute platform such as AWS EC2 or on-premise servers, and all configruation is handled by the customer. Many integrations work with SMF out of the box, and all integrations can be set up on CMF.

Reviewer: ```suggestion
Deadline Cloud fully manages the workers running the jobs and includes some pre-configured software. With CMF, the workers run on any customer-managed compute platform such as AWS EC2 or on-premise servers, and all configuration is handled by the customer. Many integrations work with SMF out of the box, and all integrations can be set up on CMF.
```

| [Maxon Cinema 4D](https://www.maxon.net/en/cinema-4d) | [deadline-cloud-for-cinema-4d](https://github.com/aws-deadline/deadline-cloud-for-cinema-4d) | ✅ |  |
| [Nuke](https://www.foundry.com/products/nuke-family/nuke) | [deadline-cloud-for-nuke](https://github.com/aws-deadline/deadline-cloud-for-nuke) | ✅ | ✅ |
| [Rhino](https://www.rhino3d.com/) | [deadline-cloud-for-rhino](https://github.com/aws-deadline/deadline-cloud-for-rhino) | ✅ |  |
| [SideFX Houdini](https://www.sidefx.com/) | [deadline-cloud-for-houdini](https://github.com/aws-deadline/deadline-cloud-for-houdini) | ✅ | ✅ |
| [Unreal Engine](https://www.unrealengine.com/) | [deadline-cloud-for-unreal-engine](https://github.com/aws-deadline/deadline-cloud-for-unreal-engine) | ✅ |  |

## Other notable repositories

| Repository | Description |
| - | - |
| [deadline-cloud](https://github.com/aws-deadline/deadline-cloud) | A Python CLI and library for interacting with Deadline Cloud, especially submitting jobs. | 
| [deadline-cloud-samples](https://github.com/aws-deadline/deadline-cloud-samples) | Example code and tools for running workloads on Deadline Cloud. | 

> | [deadline-cloud-worker-agent](https://github.com/aws-deadline/deadline-cloud-worker-agent) | Software that runs on workers and communicates with the Dealine API for receiving and completing tasks. | 

Reviewer: ```suggestion
| [deadline-cloud-worker-agent](https://github.com/aws-deadline/deadline-cloud-worker-agent) | Software that runs on workers and communicates with the Deadline Cloud API for receiving and completing tasks. | 
```
